<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>970</width>
    <height>624</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="upperLayout">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QPushButton" name="buttonEditItem">
            <property name="text">
             <string>Bearbeiten...</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="buttonDeleteItem">
            <property name="text">
             <string>Löschen</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QCheckBox" name="buttonEnableSort">
            <property name="text">
             <string>sortierbar</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>378</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="buttonInsertItem">
          <property name="text">
           <string>Hinzufügen...</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QTableWidget" name="tableTasks">
        <property name="minimumSize">
         <size>
          <width>900</width>
          <height>450</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="layoutLowerPanel">
        <item>
         <widget class="QLineEdit" name="searchText">
          <property name="minimumSize">
           <size>
            <width>125</width>
            <height>25</height>
           </size>
          </property>
          <property name="text">
           <string>Suche...</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="buttonFind">
          <property name="text">
           <string>Suchen</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>208</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <layout class="QHBoxLayout" name="layoutPathToFile">
          <item>
           <widget class="QLabel" name="labelPathToFile">
            <property name="text">
             <string>Pfad zur Datei:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="pathToFile">
            <property name="minimumSize">
             <size>
              <width>300</width>
              <height>25</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="buttonSaveList">
            <property name="text">
             <string>Speichern</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QLabel" name="labelFileName">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>970</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuListe">
    <property name="title">
     <string>Liste</string>
    </property>
    <addaction name="actionNewList"/>
    <addaction name="actionOpenList"/>
    <addaction name="actionReloadList"/>
    <addaction name="actionDeleteAllTasks"/>
   </widget>
   <addaction name="menuListe"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionOpenList">
   <property name="text">
    <string>Öffne Liste ...</string>
   </property>
  </action>
  <action name="actionNewList">
   <property name="text">
    <string>Neue Liste</string>
   </property>
  </action>
  <action name="actionReloadList">
   <property name="text">
    <string>Änderungen verwerfen</string>
   </property>
  </action>
  <action name="actionDeleteAllTasks">
   <property name="text">
    <string>Alle Aufgaben löschen</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
